seqr/views/apis/saved_variant_api.py

```diff
@@ -194,8 +194,31 @@ def delete_variant_note_handler(request, variant_guids, note_guid):
     })
 
 
+def _get_tag_type_create_data(tag, saved_variants=None):
+    variant_tag_type = VariantTagType.objects.get(
+        Q(name=tag['name']),
+        Q(project=saved_variants[0].family.project) | Q(project__isnull=True)
```

Comment:
yes but this function is being passed as the `get_tag_create_data` which has the function signature that `saved_variants` is a kwarg because its not needed by the other function passed as that argument. If you really want this can explicitly throw an error if `saved_variants` is None but its a) not testable as theres no code path that leads to that and b) it will throw an error anyways if that somehow happens so I don't think explicitly throwing an error is worth it